src/EventListener/LogRevisionsListener.php

```
@@ -308,7 +308,7 @@ private function getOriginalEntityData(object $entity): array
     }
 
     /**
-     * @return string|int
+     * @return string|int|false
```

if you're returning false, it means you're setting false to the property too.

Also `false` doesn't seems to be a supported return type because it's later use as a param in the query.

Maybe we should throw an exception if the value is false ?